src/types/products.ts

```diff
@@ -117,12 +117,19 @@ interface RootDocsPath {
 	mainBranch?: string
 }
 
+export type DocsNavItem = {
+	icon: string
+	label: string
+	fullPath: string
+}
+
 interface ProductData extends Product {
 	algoliaConfig: {
 		indexName: string
 	}
 	basePaths: string[]
 	rootDocsPaths: RootDocsPath[]
+	docsNavItems?: (DocsNavItem | string)[]
```

Comment:
`docsNavItems` is optional. If omitted (which we currently do for all products except Terraform), we fall back to using `rootDocsPaths` for the Documentation navigations in the sidebar and top nav.